### ISSUE TITLE
CDN provider addition: QUANTIL

### ISSUE THREAD

**Issue by yolafo** [igrigorik/istlsfastyet.com#94]
Hi, would it be possible to add QUANTIL to the list? Our company support the following:  

Session identifiers: yes
Session tickets: yes
OCSP stapling: no
Dynamic record sizing: no
ALPN :no
Forward secrecy : yes
HTTP/2: no

Thanks!

**Comment by igrigorik** [comment]
Yep. Do you have public documentation links to any of the "yes" features - e.g. how to enable them, etc?

**Comment by yolafo** [comment]
These are back-end configurations that are set up by our Sales Engineers. Unfortunately there is no public documentation available yet.

**Comment by igrigorik** [comment]
Hmm, OK. Can you put together a quick pull request?

**Comment by yolafo** [comment]
Hi, sorry for the delay, just set up the pull request. Have not used Github before so hopefully I did it correct. Thanks!
